### PR TITLE
Add ithinkisam recommendations

### LINKED ISSUE #9: ithinkisam checklist
- [x] Create a branch
- [x] Add a file
- [ ] Make a PR
- [ ] Merge

:us:

### PR DESCRIPTION
**Pull request by ithinkisam**
Adding recommendations for Twin Cities dining.
:burrito: was tasty. Sam owes Scott :moneybag: 

Closes #9

### CODE REVIEW

File: ithinkisam-rec.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Twin Cities, MN
+
+## Places to Eat
+-- The Tavern Grille
+-- Granite City
+-- Original Pancake House
+-- Cossetta's
```
Comment (Scott-Buermann) [inline ithinkisam-rec.md]: This is my first choice.

File: ithinkisam-rec.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Twin Cities, MN
+
+## Places to Eat
+-- The Tavern Grille
+-- Granite City
```
Comment (Scott-Buermann) [inline ithinkisam-rec.md]: This is my 2nd choice!

File: ithinkisam-rec.md (hunk 1 above)
Comment (Scott-Buermann) [inline ithinkisam-rec.md]: 1st

**Review by Scott-Buermann — COMMENTED** [review]
@ithinkisam hope it was not a "small" :burrito:

File: ithinkisam-rec.md (hunk 2 above)
Comment (ithinkisam) [reply ithinkisam-rec.md]: @Scott-Buermann I added this as 2nd choice

File: ithinkisam-rec.md (hunk 1 above)
Comment (ithinkisam) [reply ithinkisam-rec.md]: @Scott-Buermann I added this as 1st choice

**Review by Scott-Buermann — APPROVED** [review]
well done!

**Pull request merged**